CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.6
+---
+
+ * Deprecated passing more than one attribute to `@IsGranted()`. Use an expression instead
+   (e.g. `@IsGranted("has_role('ROLE_USER') or has_role('ROLE_ADMIN')")`).
```

Comment:
```suggestion
   (e.g. `@Security("has_role('ROLE_USER') or has_role('ROLE_ADMIN')")`).
```
`IsGranted` doesn't work with an expression (because you would need to pass an `Expression` object).